activiti-engine/src/main/java/org/activiti/engine/impl/bpmn/deployer/BpmnDeployer.java

```diff
@@ -72,7 +79,7 @@ public void deploy(DeploymentEntity deployment,
                 parsedDeployment.getAllProcessDefinitions());
         bpmnDeploymentHelper.setResourceNamesOnProcessDefinitions(parsedDeployment);
 
-//    createAndPersistNewDiagramsIfNeeded(parsedDeployment);
+        createAndPersistNewDiagramsIfNeeded(parsedDeployment);
```

Comment:
@daisuke-yoshimoto let's remove this from here. The engine doesn't need to provide this functionality.